### ISSUE TITLE
"Vorherige Suchanfrage löschen"-Button im Suchfeld ergänzen

### ISSUE THREAD

**Issue by vossviola** [UB-Mannheim/malibu#157]
In einigen Suchmaschinen gibt es am Ende des Suchfeldes ein Kreuzchen-Icon, mit dem die vorher eingetragene Suchanfrage gelöscht und der Cursor ins Suchfeld gesetzt wird.
Ein Beispiel: [Katalog der ULB Münster](https://katalogplus.uni-muenster.de/discovery/search?query=any,contains,women%20fire%20dangerous%20things&tab=Everything&search_scope=MyInst_and_CI&vid=49HBZ_ULM:VU2&offset=0).
Das fände ich für malibu auch praktisch, weil das bei mehreren Anfragen hintereinander einiges an Klicks bzw. Deletes sparen helfen könnte.

**Comment by zuphilip** [comment]
Ja, könnte man prinzipiell schon machen. Das wäre dann wohl bei den beiden Eingabefelder für die ISBN und die PPN bei der Suche sinnvoll. Konkret sieht diese Lösung über einen reset-Button und etwas CSS passend aus: https://stackoverflow.com/questions/2803532/how-do-i-put-a-clear-button-inside-my-html-text-input-box-like-the-iphone-does#answer-49190282

**Comment by zuphilip** [comment]
Die einfache Lösung mit `<input id="eingabeIsbn" type="search"` ist jetzt Teil des PR#169. Daher haben wir jetzt diese Funktionalität für Chrome und andere darauf basierende Browser z.B. Edge. Leider unterstützt dies Firefox nicht auch, aber hier scheint mir eine Speziallösung unnötig kompliziert zu sein.
